### ISSUE TITLE
[a11y] Schema editor doesn't work 

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1319]
**Describe the bug**
No schema change saves in the Locality table

**To Reproduce**
1. Go to Schema Conifg
2. Change text1's name and description
3. Save
4. Reload and see error

macOS, Edge

**Comment by maxxxxxdlp** [comment]
Is this fixed if you also clean the cache?

**Comment by maxxxxxdlp** [comment]
Fixed in [29dd573](https://github.com/specify/specify7/commit/29dd57357a10560b299a41366216d92095ec1766)
